samples/highcharts/sonification/chart-earcon/demo.js

```diff
@@ -54,5 +65,15 @@ var chart = Highcharts.chart('container', {
 
 
 document.getElementById('sonify').onclick = function () {
+    document.getElementById('sonify').style.visibility = 'hidden';
+    document.getElementById('overlay').style.visibility = 'hidden';
+    document.getElementById('stop').style.visibility = 'visible';
     chart.sonify();
```

Comment:
Could we set focus to the stop button when clicking the sonify button? `getElementById('stop').focus()`